### PR TITLE
Add Establish Claim Request

### LINKED ISSUE #122: Add `establish_claim` API method
Need to test with the proper EP codes - likely EP170 & EP172

### PR DESCRIPTION
**Pull request by shanear**
Since the URL for this endpoint is different for the eFolder endpoints, it
requires the use of `CONNECT_VBMS_BASE_URL` environment variable instead of
`CONNECT_VBMS_URL`.

fixes #122

### CODE REVIEW

File: lib/vbms/common.rb [hunk 1]
```diff
@@ -29,7 +29,13 @@ module VBMS
...
     wsu: "http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd",
     ds: 'http://www.w3.org/2000/09/xmldsig#',
-    xenc: 'http://www.w3.org/2001/04/xmlenc#'
+    xenc: 'http://www.w3.org/2001/04/xmlenc#',
+    claimV4: "http://vbms.vba.va.gov/external/ClaimService/v4"
+  }.freeze
+
+  ENDPOINTS = {
+    claims: "/vbmsp2-cs/ClaimServiceV4",
+    efolder: "/vbmsp2-cms/streaming/eDocumentService-v4"
```
Comment (askldjd) [inline lib/vbms/common.rb]: Subjective. 

With this naming, I feel like we have a slight dependency inversion issue. `connect_vbms` is a lower level module, and shouldn't have awareness to high level application such as `efolder`.

File: lib/vbms/requests/base_request.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module VBMS
+  module Requests
+    # Abstract class providing defaults to some of the methods requried
+    class BaseRequest
+      def inject_header_content(xml)
+        xml
+      end
+    end
```
Comment (askldjd) [inline lib/vbms/requests/base_request.rb]: Seems like most request has `multipart=false`. Maybe that should be the base class default.

File: lib/vbms/responses/claim.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  module Responses
+    class Claim
+      attr_accessor :claim_id
+  
+      def initialize(claim_id: nil)
+        self.claim_id = claim_id
+      end
+  
+      def self.create_from_xml(el)
+        new(claim_id: el["id"])
```
Comment (askldjd) [inline lib/vbms/responses/claim.rb]: How do we handle SOAPFault?

File: lib/vbms/requests/base_request.rb (hunk 2 above)
Comment (shanear) [inline lib/vbms/requests/base_request.rb]: i thought about that, but i came to the conclusion that that was sort of confusing. Now that you say this im coming around to it.

File: lib/vbms/common.rb (hunk 1 above)
Comment (shanear) [inline lib/vbms/common.rb]: This is the url for VBMS efolder service, not efolder express. We could make these values configuration variables, but it seems like a pain. What do you think?

File: lib/vbms/responses/claim.rb (hunk 3 above)
Comment (shanear) [inline lib/vbms/responses/claim.rb]: https://github.com/department-of-veterans-affairs/connect_vbms/blob/master/lib/vbms/client.rb#L87

SOAPFaults are not 200's correct?

File: lib/vbms/requests/base_request.rb (hunk 2 above)
Comment (shanear) [inline lib/vbms/requests/base_request.rb]: made the change

**Pull request merged**